src/components/engine-connector/engine-connector.js

```diff
@@ -36,7 +36,7 @@ class EngineConnector extends Component {
 
     eventManager.listen(player, player.Event.PLAYER_RESET, () => {
       this.props.updateCurrentTime(0);
-      this.props.updateIsIdle(true);
+      // this.props.updateIsIdle(true);
```

Comment:
see my other comment

Reply:
all the required changes in the side panel side have already been made 

As detailed in the description, the issue is caused by the playkit-ui and not the plugin side because of the transition between the presets

and regarding that https://github.com/kaltura/playkit-js-ui/pull/574